### DISCUSSION TITLE
Internal Error  WindowBuilder encountered unexpected internal error.   This could be caused by a WindowBuilder bug or by a misconfiguration issue, conflict, partialupdate, etc.    Show stack trace.

### DISCUSSION THREAD

**Discussion by visalvisale** [eclipse/windowbuilder#719] [Q&A]
Internal Error

WindowBuilder encountered unexpected internal error. 

This could be caused by a WindowBuilder bug or by a misconfiguration issue, conflict, partialupdate, etc.

[Show stack trace.](javascript:toggleVisibleAll();)

**Comment by ptziegler** [comment]
What is the stack trace? 

Can you perhaps also post a screenshot of your workspace and/or the source code?
